transformations/random_walk/transformation.py

```diff
@@ -0,0 +1,215 @@
+#import itertools
+import random
+import numpy as np
+import re
+import copy
+import random
+from typing import List
+
+from transformers import BertTokenizer, BertForMaskedLM
+from sentence_transformers import SentenceTransformer, util
+
+from torch.nn import functional as F
+import torch
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+'''
+def _mask_word(sentence, word_to_mask, tokenizer):
+    """ helper function,
+    replace word in a sentence with mask-token, as prep for BERT tokenizer"""
+    start_index = sentence.find(word_to_mask)
+    return sentence[0:start_index] + tokenizer.mask_token + sentence[
+        start_index + len(word_to_mask):]
+'''
+
+def _mask_word(sentence, split_indices, mask):
+    """ helper function,
+    replace word in a sentence with mask-token, as prep for BERT tokenizer"""
+    return sentence[0:split_indices[0]] + mask + sentence[
+        split_indices[1]:]
+
+
+def get_k_replacement_words(tokenized_text, tokenizer, model, k):
+    """return k most similar words from the model, for a tokenized mask-word in a sentence.
+
+    Args:
+        tokenized_text (str): sentence with a word masked out
+        model ([type]): model
+        tokenizer ([type]): tokenizer
+        k (int, optional): how many similar words to find for a given tokenized-word. Defaults to 5.
+
+    Returns:
+        [list]: list of top k words
+    """
+    inputs = tokenizer.encode_plus(tokenized_text, return_tensors='pt', truncation=True, max_length = 512)
+    index_to_mask = torch.where(inputs.input_ids[0] == tokenizer.mask_token_id)
+    if index_to_mask[0].numel() == 0: # Since we are truncating the input to be 512 tokens (BERT's max), we need to make sure the mask is in these first 512.
```

Comment:
nitpick: don't let text go over 80 characters in width (break comments into several lines, and run `yapf` or `black`)